modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -689,6 +850,59 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private Map<String, ObjectRelationship> _getAvailableObjectRelationshipMap(
+		ObjectDefinition objectDefinition) {
+
+		Map<String, ObjectRelationship> objectRelationshipMap = new HashMap<>();
+
+		List<ObjectRelationship> availableObjectRelationships =
+			_objectRelationshipLocalService.getObjectRelationships(
+				objectDefinition.getObjectDefinitionId());
+
+		ObjectRelationship objectRelationship;
+
+		if (!availableObjectRelationships.isEmpty()) {
```

Comment:
This if clause is not necessary as the for loop will not be executed in case there are no elements